icu4c/source/i18n/double-conversion-double-to-string.h

```diff
@@ -138,7 +138,7 @@ class DoubleToStringConverter {
   // Example with max_leading_padding_zeroes_in_precision_mode = 6.
   //   ToPrecision(0.0000012345, 2) -> "0.0000012"
   //   ToPrecision(0.00000012345, 2) -> "1.2e-7"
-  // Similarily the converter may add up to
```

Comment:
Note/FYI: The files that start with `double-conversion-*.h` actually come from an upstream project called ["double-conversion"](https://github.com/google/double-conversion). They are semi-regularly updated by pulling in the files from that project, which unfortunately means that this fix (and others) will eventually get overridden when they are updated... 

If you wanted to help contribute to another open-source project, (if you're so inclined) then you could maybe open a pull-request to fix them upstream. 😊

For example, this file is from here: https://github.com/google/double-conversion/blob/master/double-conversion/double-to-string.h#L126

Reply:
See https://github.com/google/double-conversion/pull/163 :wink:

Reply:
Thank you very much!